exercises/05/readme.md

```diff
@@ -214,7 +214,7 @@ curl \
 
 For Windows users, this is the equivalent command (basically you have to use double quotes throughout, and therefore some must be escaped with `\`, and the line continuation character is `^` rather than `\`):
 
-```
+```sh
```

Comment:
why the change from shell to sh (Q applies to other occurrences as well, of course :-))

Reply:
I added it here because I started with 'sh' in the other places to stay consistent. Then, I noticed that we also use `bash` in other places and gave up :D 
I don't mind how we name it, as long as it is consistent. I agree. 
Which one do you prefer?

Reply:
Fixes in PR #23